run_wml_tests

```diff
@@ -58,6 +58,11 @@ class TestListParser:
             test_list.append(t)
         return test_list
 
+def get_output_filename(args):
```

Comment:
Please raise an exception if the `for` loop finishes.

While there always should be a `-u` argument, if there's a bug somewhere else in the script then it might be missing; if this function doesn't return a filename then that will complicate debugging the other bug in the script.

Reply:
If there's no `-u` argument, then the script shouldn't have gotten to this point in the first place, I would think.

Reply:
If there's no `-u` argument then someone is making changes elsewhere in the script, maybe adding an alternative command-line option for running tests. Adding an exception would make it far more obvious what broke, because currently the error message doesn't hint about `-u`:
```
TypeError: expected str, bytes or os.PathLike object, not NoneType
```

Reply:
Updated.